pm_dftb_opt/pm_dftb_opt.xml

```diff
@@ -1,4 +1,5 @@
-<tool id="pm_dftb_opt_write" name="Generate DFTB+ outputs (dftb+)" version="0.1.0" python_template_version="3.5" profile="22.01">
+<tool id="pm_dftb_opt" name="PyMuonSuite AIRSS DFTB+ Opt" version="0.1.0" python_template_version="3.5" profile="22.01">
```

Comment:
Sorry if I wasn't clear - we shouldn't use 'opt' for user-facing stuff, it's not an established abbreviation
```suggestion
<tool id="pm_dftb_opt" name="PyMuonSuite AIRSS DFTB+ Optimise" version="0.1.0" python_template_version="3.5" profile="22.01">
```